src/core/core.scale.js

```diff
@@ -331,6 +335,30 @@ var Scale = Element.extend({
 		return me.minSize;
 
 	},
+
+	/**
+	 * @private
+	 */
+	_configure: function() {
```

Comment:
It would be nice to also set `me._isHorizontal` here and just return that in `me.isHorizontal()` and use it directly instead of calling `me.isHorizontal()`

Reply:
Its used a lot before first `_configure` call. Would have to do it in `initialize`. Most likely negligible effect, so might not be worth the effort.